### ISSUE TITLE
404 pages show up with link AMPHTML tag inserted.

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#2018]
<link rel="amphtml" href="https://www.mydomain/blog/somepost/amp/" /> being injected into error 404 pages. Appears to be the url of the last blog post. 

Situation is a regular site and NOT in AMP takeover.

**Comment by Sejiro-Hiko** [comment]
@Frenchomatic I don't see the amphtml tag on my end on 404 pages or am I looking at the wrong place?

**Comment by Frenchomatic** [comment]
It seems to happen only when a blog is defined. 

```
<html lang="fr-FR">
<head>
<meta charset="UTF-8" />
<meta name="robots" content="noodp,noydir" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Page non trouvée</title>
<link rel="amphtml" href="https://www.mydomain.fr/blog/initiation-au-ski-de-rando/amp/" /><link rel='dns-prefetch' href='//s.w.org' />
```

I flick over the real site by email right now so you can see

**Comment by Sejiro-Hiko** [comment]
@Frenchomatic  I've checked the site and yeah it's a strange issue also I've replied you back to your email have a look at that mail.

**Comment by Frenchomatic** [comment]
sent you login details so you can have a poke around. Let me know when you are done.

**Comment by Frenchomatic** [comment]
Looks like you fixed it but now it is embedding a rel="next" .  I sent an email as to potentially why it is doing that - can you please let me know what files you changed so I can go to 9.5.8.5 and do my usual mods.

**Comment by Sejiro-Hiko** [comment]
@Frenchomatic please check the email which I've sent you.

**Comment by Frenchomatic** [comment]
yes i thin we fixed it by using this in the 404.php genesis template

`remove_action( 'wp_head', 'genesis_paged_rel' );  // THIS STOPS re="next" being put into error 404 pages by the AMP plugin
`
